google/cloud/aiplatform/initializer.py

```diff
@@ -213,7 +213,12 @@ def project(self) -> str:
     @property
     def location(self) -> str:
         """Default location."""
-        return self._location or constants.DEFAULT_REGION
+        return (
+            self._location
+            # Inferring the location from Google Cloud Vertex AI runtime environment
+            or os.environ.get("CLOUD_ML_REGION")
```

Comment:
Please centralize to https://github.com/googleapis/python-aiplatform/blob/main/google/cloud/aiplatform/training_utils/environment_variables.py and reference.